Sources/ShortcutFoundation/Extensions/Int+Extension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Int+Extension.swift
+//  ShortcutFoundation
+//
+//  Created by Andreas Lif on 2022-01-12.
+//  Copyright © 2021 Shortcut Scandinavia Apps AB. All rights reserved.
+//
+
+import Foundation
+
+public extension Int {
+    
+    var isEven: Bool {
+        guard self != 0 else { return true }
```

Comment:
@AndreasShortcut 

Could we try:

return self.isMultiple(of: 2)

Reply:
Yes, that's maybe easier to understand. Fixed in 5fcd885